tests/rptest/tests/flink_basic_test.py

```diff
@@ -88,11 +71,113 @@ def test_basic_workload(self):
                 self.logger.warning(f"Job '{_id}' has failed")
                 _failed.append(_job)
 
+        return _failed
+
+    def _get_workloads_by_tags(self, tags):
+        """
+            Calls to manager to get workloads using supplied list of tags
+            Raises error on nothing found
+
+            return: list of workloads
+        """
+        workloads = self.workload_manager.get_workloads(tags)
+        if len(workloads) < 1:
+            raise RuntimeError("No workloads found "
+                               f"with tags: {', '.join(tags)}")
+        return workloads
+
+    @cluster(num_nodes=4)
+    def test_basic_workload(self):
+        """
+            Test starts produce workload and then consume
+            No checks for message counts, just job success
+        """
+
+        # Start Flink
+        self.flink.start()
+
+        # Load python workload to target node
+        # Hardcoded file
+        # TODO: Add workload config management
+        _workload_config = {
+            "log_level": "DEBUG",
+            "brokers": self.redpanda.brokers(),
+            "producer_group": "flink_produce_group",
+            "consumer_group": "flink_consume_group",
+            "topic_name": self.topic_name,
+            "msg_size": 4096,
+            "count": 10
+        }
+
+        # Run produce basic
+        workloads = self._get_workloads_by_tags(['flink', 'produce', 'basic'])
+        _failed = self._run_workloads(workloads, _workload_config)
+
+        # Assert failed jobs
+        assert len(_failed) == 0, \
+            f"Flink reports failed jobs for basic produce workloads {_failed}"
+
+        # Run consume basic
+        workloads = self._get_workloads_by_tags(['flink', 'consume', 'basic'])
+        _failed = self._run_workloads(workloads, _workload_config)
+
+        # Assert failed jobs
+        assert len(_failed) == 0, \
+            f"Flink reports failed jobs for basic consume workload {_failed}"
+
         # Stop flink
         self.flink.stop()
 
+        return
+
+    @cluster(num_nodes=4)
+    def test_transaction_workload(self):
```

Comment:
I ran this test locally and I don't see any transactions against redpanda, am I doing something wrong? There are no logs in buildkite to verify the CI run.

```
RP_TRIM_LOGS=false DUCKTAPE_REPEAT=1 REDPANDA_LOG_LEVEL=trace ENABLE_GIT_HASH=OFF BUILD_TYPE=release task rp:run-ducktape-tests DUCKTAPE_ARGS="tests/rptest/tests/flink_basic_test.py::FlinkBasicTests.test_transaction_workload"
```

Reply:
There should be local version of flink workload config. Here is my example. I am running RP with docker compose locally
```
{
    "connector_path": "file:///Users/savex/projects/flink/connectors/flink-sql-connector-kafka-3.0.1-1.18.jar",
    "logger_path": "/Users/savex/projects/flink",
    "data_path": "file:///Users/savex/projects/flink/data",
    "log_level": "DEBUG",
    "brokers": "localhost:19092",
    "producer_group": "flink_produce_group",
    "consumer_group": "flink_consume_group",
    "topic_name": "flink_transactions_topic",
    "mode": "consume",
    "word_size": 32,
    "batch_size": 16,
    "count": 128
}
```

Reply:
Please, can you send me the test.debug from local run?

Reply:
https://gist.github.com/bharathv/1f738d67fb1ea89a86bce96cc1cf233a

Also there are no flink service logs from what I can tell.

Reply:
Just did a double check with breakpoint at `flink.stop()`
```
ubuntu@ip-172-31-61-216:~/tests/results$ rpk --brokers ip-172-31-6-120:9092,ip-172-31-9-198:9092,ip-172-31-1-33:9092 topic consume flink_workload_topic
{
  "topic": "flink_workload_topic",
  "value": "\"rHVAALOlYhDEoBBqBADZqiaebXokPrIFPp\"",
  "timestamp": 1704483918866,
  "partition": 0,
  "offset": 0
}
{
  "topic": "flink_workload_topic",
  "value": "\"fMkHaEYitbSgLvaPCrIQzAzPNViKOVKYmuGBArBafVldc\"",
  "timestamp": 1704483918879,
  "partition": 0,
  "offset": 1
}
....
```

Reply:
Did something change? I still see the same behavior. You don't see it locally with 

```
RP_TRIM_LOGS=false DUCKTAPE_REPEAT=1 REDPANDA_LOG_LEVEL=trace ENABLE_GIT_HASH=OFF BUILD_TYPE=release task rp:run-ducktape-tests DUCKTAPE_ARGS="tests/rptest/tests/flink_basic_test.py::FlinkBasicTests.test_transaction_workload"
```